.github/workflows/cd.yml

```diff
@@ -68,6 +76,9 @@ jobs:
         target: runtime
         context: .
         file: ./docker/Dockerfile
+        platforms: |
+          linux/amd64
+          linux/arm64
```

Comment:
x2

As it's basically re-building the image. It also uses the cache, so it won't be double the time in those cases.

That's from 30 to 60 minutes, without cache.

Reply:
Could we disable the ARM build by default?

That seems like a lot of time for a platform that isn't our primary one.
How likely is it that amd64 will succeed, but arm64 will fail?

Reply:
The only _bad_ thing of not building with arm64 is if anyone wants to test zebra on an M1/Raspberry or something similar without having to install Rust and all the required tooling.

If we add new crates which won't build with arm64, that's a reason to fail.

An option is to just build this arm64 compatible image when pushing to main, and not to test every PR against arm64. This wouldn't have a direct impact on the development process.

Reply:
> An option is to just build this arm64 compatible image when pushing to main, and not to test every PR against arm64. This wouldn't have a direct impact on the development process, but it would allow people to have a ARM64 version to test/develop with.

This seems like a good compromise.

Reply:
Thanks!